src/SignUpIn/DelivererGroup/DelivererGroupSignUpController.js

```diff
@@ -0,0 +1,99 @@
+import '../SignUpIn.css';
+import DelivererGroupSignUp from './DelivererGroupSignUp';
+import DelivererGroupSignUp1 from './DelivererGroupSignUp1';
+import DelivererGroupSignUp2 from './DelivererGroupSignUp2';
+import SignUpComplete from '../SignUpComplete';
+
+let React = require('react');
```

Comment:
same as above
please change to ES6 format
line 7, 8 27-89
https://daveceddia.com/react-es5-createclass-vs-es6-classes/